frontend/README.md

```diff
@@ -33,8 +33,6 @@ In `frontend.env`, add:
 
 > NOTE: The `VITE_BACKEND_API_BASE_URL` environment variable is the base url endpoint that the backend is running on. If the environment variable is not specified, the react application will default to using `http://localhost:8000/` as the base url when calling the API endpoint.
 
-For features that are still in progress, we have a boolean flag which enables these features to be shown while developing. To enable this, simply add `VITE_ENV=dev` to `frontend.env`. 
```

Comment:
Why is this removed

Reply:
We can check if the app is in dev (and enable in dev features) via one of vite's default env vars https://vitejs.dev/guide/env-and-mode.html#env-variables, rather than using our own custom env vars (just realised we could do this LOL)